css-js/fixup.js

```diff
@@ -0,0 +1,229 @@
+/******************************************************************************
+ *                 JS Extension for the W3C Spec Style Sheet                  *
+ *                                                                            *
+ * This code handles:                                                         *
```

Comment:
Is the file a copy and modified version of a standard ReSpec file? If so, are the changed elements worth noting?

Reply:
direct copy, added URI and date